src/Client/DurableStorage/FilesystemDurableStorage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+namespace Tuf\Client\DurableStorage;
+
+/**
+ * Class FilesystemLocalState
+ *
+ * A simple implementation of LocalStateInterface using the filesystem.
+ * Applications might want to provide an alternative implementation with better performance and error handling.
```

Comment:
We could also add a PHPCS rule from that (borrow Drupal's?).

Reply:
If we're gonna do it we should add phpcs for it. Can we add that as an additional style-only & associated phpcs rule change to #19 to keep PRs more clean?

Reply:
+1 to making it a separate PR.

Reply:
Fixing it in #19

Reply:
Hmm, having phpcs enforce line length only for comments to 80 doesn't seem to be a "just turn it on" kind of thing. I'm looking in Drupal's Coder module now to see if there's some custom sniff that does it...

PSR-2 recommends 120 chars and is explicit that it is not a hard requirement.

Reply:
This is fixed now in the branch and we have #26 to discuss the overall question of comment line length warnings, so marking resolved since the rest is out of scope. Thanks!